### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by dslomov**
Adding GitHub repo admins to global CODEOWNERS

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @PiotrSikora @martinmaly
```
Comment (qiwzhang) [inline CODEOWNERS]: could you add my name too: qiwzhang

Comment (martinmaly) [inline CODEOWNERS]: and I'd recommend removing me since I no longer work on this project

Comment (PiotrSikora) [reply CODEOWNERS]: Same here (though, you can leave me in addition to @qiwzhang if you need at least 2 people).

Comment (qiwzhang) [reply CODEOWNERS]: @PiotrSikora we need you. Please stay

**Pull request merged**